client/src/components/UsedOnTable/UsedOnTable.js

```diff
@@ -64,23 +63,24 @@ class UsedOnTable extends PureComponent {
       id,
       title,
       type,
-      state,
       link,
     } = data;
 
-    const badge = (state)
-      ? <span className={classnames('badge', 'used-on__badge', `status-${state}`)}>{state}</span>
-      : null;
-
     const titleLabel = (link)
-      ? <a className="used-on__edit-link" href={link}>{title} {badge}</a>
-      : <span>{title} {badge}</span>;
+      ? <a className="used-on__edit-link" href={link}>{title}</a>
+      : <span>{title}</span>;
+
+    const handleTrClick = (event) => {
+      event.target.querySelector('a').click();
+    };
 
     return (
-      <tr key={id}>
-        <td className="used-on__col--index">{index + 1}</td>
-        <td className="used-on__col--title">{titleLabel}</td>
-        <td className="used-on__col--type">{type}</td>
+      <tr key={id} className="used-on__row" onClick={handleTrClick}>
```

Comment:
I think you also need to give `role=button` style attributes when making things like this clickable (for accessibility)